push_notification/common/models/base_entity.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+
+
+class BaseEntity(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
```

Comment:
Naming can be better since `BaseEntity` looks a lot generic and providing less information about its functionality.